R/partial_correlation_influence.R

```diff
@@ -3,4 +3,4 @@
 #partial correlation analysis of the stock market. PLoS ONE 5, e15032 (2010).
```

Comment:
We should have R source filenames be in the format `name-other.R`.  In this case `random.R` and `partial-correlation-influence.R`.